<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ATrackDrawProperties</class>
 <widget class="QDialog" name="ATrackDrawProperties">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>306</width>
    <height>169</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>2</y>
     <width>300</width>
     <height>131</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>281</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>198</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Photon track attributes:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbPhotons_norm">
    <property name="geometry">
     <rect>
      <x>215</x>
      <y>6</y>
      <width>78</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Show / edit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbPhotons_PM">
    <property name="geometry">
     <rect>
      <x>215</x>
      <y>33</y>
      <width>78</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Show / edit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbPhotons_sec">
    <property name="geometry">
     <rect>
      <x>215</x>
      <y>60</y>
      <width>78</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Show / edit</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbSkipPhotonsMissingPMs">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>90</y>
      <width>283</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Do not build tracks for photons missing PMs</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbSpecialRule_hitPM">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>36</y>
      <width>191</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>For photons hitting PMs:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbSpecialRule_secScint">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>62</y>
      <width>191</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>For photons of secondary scint:</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pbClose">
   <property name="geometry">
    <rect>
     <x>78</x>
     <y>140</y>
     <width>151</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>pbPhotons_norm</tabstop>
  <tabstop>cbSpecialRule_hitPM</tabstop>
  <tabstop>pbPhotons_PM</tabstop>
  <tabstop>cbSpecialRule_secScint</tabstop>
  <tabstop>pbPhotons_sec</tabstop>
  <tabstop>cbSkipPhotonsMissingPMs</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
